ft_volumenormalise.m

```diff
@@ -137,6 +138,7 @@
     if strcmpi(cfg.spmversion, 'spm2'),  cfg.template = fullfile(spmpath, filesep, 'templates', filesep, 'T1.mnc'); end
     if strcmpi(cfg.spmversion, 'spm8'),  cfg.template = fullfile(spmpath, filesep, 'templates', filesep, 'T1.nii'); end
     if strcmpi(cfg.spmversion, 'spm12'), cfg.template = fullfile(spmpath, filesep, 'toolbox',   filesep, 'OldNorm', filesep, 'T1.nii'); end
+    cfg.templatecoordsys = 'spm';
```

Comment:
it is not needed to specify it here right? It is already specified on line 110, with a default value of 'spm'

Reply:
This will override the cfg.templatecoordsys provided when using spm's templated to make sure it is consistent with what is done.